.github/workflows/owasp-dependency-check-daily.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
   analyze:
     runs-on: ubuntu-latest
-
+    permissions: write-all
```

Comment:
I don't think we need the permission on this part
```suggestion

```

Reply:
let me run it again without it. the other one is green